consensus/common/src/main/java/org/hyperledger/besu/consensus/common/PoaQueryServiceImpl.java

```diff
@@ -46,4 +52,9 @@ public Address getProposerOfBlock(final BlockHeader header) {
   protected Blockchain getBlockchain() {
     return blockchain;
   }
+
+  @Override
+  public Address getLocalSignerAddress() {
+    return org.hyperledger.besu.ethereum.core.Address.extract(localNodeKeypair.getPublicKey());
```

Comment:
you may get away by static importing `extract` method. Not a show stopper.

Reply:
We've got two `Address` classes in scope - one for Besu core, one for plugin API.  I've left this one as an explicit reference to aid with context.